test/blackbox-tests/test-cases/melange/empty-entries-target-dir-exists.t

```diff
@@ -17,5 +17,9 @@ Create the dist folder
   $ mkdir ./dist
 
   $ dune build @melange-dist
+  Error: No rule found for dist/melange.js
```

Comment:
@jchavarri i don't understand how this test worked before. Where does the `melange.js` file come from?

Reply:
I don't know either. @anmonteiro added the test in https://github.com/ocaml/dune/pull/6506 and then modified it in https://github.com/ocaml/dune/pull/6579/.

Reply:
I’m not sure where it comes from either. I assumed it was the empty wrapper when entries are empty

Reply:
> I assumed it was the empty wrapper when entries are empty

That's indeed the reason. Perhaps we should give a more obvious name.

Anyway this caught a bug which I now fixed.